beginner_source/Intro_to_TorchScript_tutorial.py

```diff
@@ -274,6 +274,8 @@ def forward(self, x, h):
 
 my_cell = MyCell(MyDecisionGate())
 traced_cell = torch.jit.trace(my_cell, (x, h))
+
+print(traced_cell.dg.code)
```

Comment:
Suggestion: Add some text here and similarly below, to say which `forward` is for the decision gate and which is for the cell